admin/routes/restart.js

```diff
@@ -5,10 +5,12 @@ const router = express.Router()
 
 const {
   getRestartOverview,
-  getSelectRestartList
+  getSelectRestartList,
+  postSubmitRestartList
 } = require('../controllers/restart')
 
 router.get('/overview', isAuthenticated(config.ROLE_TEACHER), (req, res) => getRestartOverview(req, res))
```

Comment:
overview seems a bit vague - can we go with /restarts/overview or /restarts-overview please

Reply:
@GuyHarwood on app.js I prefix all these routes with /restart
line 221: app.use('/restart', restart)
So essentially this is going to be /restart/overview